src/models/user.js

```diff
@@ -4,9 +4,17 @@ export default {
   state: {
     favoriteControllers: [],
     favoriteResults: [],
+    user: {},
```

Comment:
We should avoid naming state variables that have the same names as the model namespace in order to improve our store's organization.